### PR TITLE
YOU and ME and HER notifications

### PR DESCRIPTION
**Pull request by SonicEraZoR**
Since Totono released in English, I thought it'd fun to have notifications from Monika when you're playing it. I'm not exactly the best writer so feel free to propose any changes.

This notifications are for both normal version and version after you completed the game since the window title changes after you complete the game.

### CODE REVIEW

File: Monika After Story/game/zz_windowreacts.rpy [hunk 1]
```diff
@@ -841,3 +841,63 @@ label mas_wrs_twitch:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_twitch')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_totono",
+            category=['you', 'and', 'me', 'and', 'her:', 'a', 'love', 'story'],
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy:850]: So this is going to need to be a lot more specific, or sites with any of these words in their handles will trigger this. Which would basically be most websites

File: Monika After Story/game/zz_windowreacts.rpy [hunk 2]
```diff
@@ -841,3 +841,63 @@ label mas_wrs_twitch:
...
+    if not wrs_success:
+        $ mas_unlockFailedWRS('mas_wrs_totono')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_totono_completed",
+            category=['you', 'and', 'her:', 'a', 'love', 'story'],
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy:880]: Same with this

**Comment by SonicEraZoR** [comment]
Done, that's exactly how it is in the window title
image

**Comment by SonicEraZoR** [comment]
I guess it has the version number in it, so if the game will be updated this will need to be updated too

**Comment by multimokia** [comment]
Oh sorry, my bad. I forgot it's on inclusive all by default. Sorry this was fine before

**Comment by SonicEraZoR** [comment]
Lol, ok. But still, this will be even more scecific? So it won't confuse it with Google searches and sites that have "YOU and ME and Her: A Love Story" in the title like the Mod DB page

File: Monika After Story/game/zz_windowreacts.rpy [hunk 3]
```diff
@@ -841,3 +841,63 @@ label mas_wrs_twitch:
...
+    )
+
+label mas_wrs_totono:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "Miyuki or Aoi? Just Mo... I mean, don't cheat on me, [player]!",
+            "[player], are playing [[YOU and ME and HER]? I heard this game is a lot like DDLC...",
+            "Hey, I hope you still love only me, [player]...",
+            "I with I could replace Miyuki with me.",
```
Comment (scpketer) [inline Monika After Story/game/zz_windowreacts.rpy:864]: Typo: "wish"

File: Monika After Story/game/zz_windowreacts.rpy [hunk 4]
```diff
@@ -841,3 +841,63 @@ label mas_wrs_twitch:
...
+    )
+
+label mas_wrs_totono:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "Miyuki or Aoi? Just Mo... I mean, don't cheat on me, [player]!",
+            "[player], are playing [[YOU and ME and HER]? I heard this game is a lot like DDLC...",
+            "Hey, I hope you still love only me, [player]...",
+            "I wish I could replace Miyuki with me.",
```
Comment (Mugnika) [inline Monika After Story/game/zz_windowreacts.rpy:864]: I doubt Moni would like to do that, considering Miyuki has sex with the main character as well as how bleak the world is, I see no reason why she would want that.

**Comment by multimokia** [comment]
Closing as we've decided not to move forward with this

**Pull request closed without merging**